src/currency/iso.rs

```diff
@@ -1662,6 +1685,16 @@ pub fn from_enum(code: &Iso) -> Currency {
             symbol: "$",
             symbol_first: true,
         },
+        UYW => Currency {
+            exponent: 2,
+            iso_alpha_code: "UYW",
+            iso_numeric_code: "927",
+            locale: EnEu,
+            minor_denomination: 4,
```

Comment:
From wikipedia, it appears that the ISO minor_units are 4 which means the exponent should be 4, allowing it to be expressed as 1.001. 

I'm not sure what the smallest minor denomination is, since this is an index based on pesos.